app/storage/s3NewClient.js

```diff
@@ -80,6 +80,19 @@ module.exports = class S3NewClient {
     }).promise();
   }
 
+  async deleteMultipleObjects(bucketName, keys) { // array of string keys
```

Comment:
```suggestion
  async deleteObjects(bucketName, keys) { // array of string keys
```

Reply:
Do we need any type checking to ensure that keys is an Array?

Reply:
the reason this function and all other functions in this file do not validate input parameters is because they all have only one client, used only internally and do not accept unknown input.